gp-nested-forms/gpnf-triggered-population.php

```diff
@@ -76,10 +76,10 @@ public function output_script() {
 
 						$form = $( '#gform_{0}'.format( self.formId ) );
 
-						$( '#field_{0}_{1}'.format( self.formId, self.triggerFieldId ) ).find( 'input' ).on( 'change', function() {
+						$( '#field_{0}_{1}'.format( self.formId, self.triggerFieldId ) ).find( ':input' ).on( 'change', function() {
 							var input = $( this );
 							var value = input.val();
-							var checked = input[0].checked;
+							var checked = ( input[0].nodeName === 'SELECT' && self.triggerFieldValue === value ) ? true : input[0].checked;
```

Comment:
Is this a remnant from the other PR that is for adding support for selects?

Reply:
Got it. Just to confirm, if the trigger field value does not match what's configured, does it remove the child entry?

Reply:
It does.